gpjax/sklearn/strategy.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class AbstractStrategy:
+    pass
+
+
+@dataclass
+class ExactInference(AbstractStrategy):
+    pass
+
+
+@dataclass
+class VariationalInference(AbstractStrategy):
+    pass
+
+
+@dataclass
+class MCMCInference(AbstractStrategy):
+    pass
```

Comment:
Is this file used?

Reply:
Not yet. Tbh, accidentally committed this.